Lib/idlelib/config_key.py

```diff
@@ -219,33 +224,34 @@ def TranslateKey(self, key, modifiers):
         return key
 
     def OK(self, event=None):
-        if self.advanced or self.KeysOK():  # doesn't check advanced string yet
-            self.result=self.keyString.get()
+        key = self.keyString.get()
+        key.strip()
```

Comment:
This is a bug.  See why?  I noticed while writing a test for showerror() below and the new test fails without the fix.